app/src/main/java/net/grandcentrix/blutufintegration/di/Modules.kt

```diff
@@ -0,0 +1,22 @@
+package net.grandcentrix.blutufintegration.di
+
+import net.grandcentrix.blutufintegration.data.repo.BluetoothRepository
+import net.grandcentrix.blutufintegration.ui.detail.DetailViewModel
+import net.grandcentrix.blutufintegration.ui.list.ListViewModel
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.dsl.module
+
+val applicationModule = module {
+
+    viewModel {
+        ListViewModel(get())
```

Comment:
Please use named parameter here. Otherwise it is not visible what `get()` actually injects without having a look to the code in an IDE like IntelliJ which has a parameter overlay.

Reply:
please also change the parameter to `bluetoothRepository = get()` here.

Reply:
yes, sry that was missing.